### ISSUE TITLE
ERR_REQUIRE_ESM when run script.

### ISSUE THREAD

**Issue by gzll** [jonluca/har-to-openapi#7]
Typescript
```
import { generateSpec } from "har-to-openapi";
import type { Har } from "har-format";

const test = async () => {
	const openapi = await generateSpec({"log":{}} as Har);
}
console.log('test');
```
when run this ts script, node report error:

```
\WorkShop\nodejs\har2openapi\node_modules\har-to-openapi\dist\quicktype.js:9
const lodash_es_1 = require("lodash-es");
                    ^

Error [ERR_REQUIRE_ESM]: require() of ES Module \WorkShop\nodejs\har2openapi\node_modules\lodash-es\lodash.js from \WorkShop\nodejs\har2openapi\node_modules\har-to-openapi\dist\quicktype.js not supported.
Instead change the require of lodash.js in \WorkShop\nodejs\har2openapi\node_modules\har-to-openapi\dist\quicktype.js to a dynamic import() which is available in all CommonJS modules.    at Object.<anonymous> (\WorkShop\nodejs\har2openapi\node_modules\har-to-openapi\dist\quicktype.js:9:21)
    at Object.<anonymous> (\WorkShop\nodejs\har2openapi\node_modules\har-to-openapi\dist\helpers.js:8:21)
    at Object.<anonymous> (\WorkShop\nodejs\har2openapi\node_modules\har-to-openapi\dist\index.js:10:19)
    at Object.<anonymous> (\WorkShop\nodejs\har2openapi\test.js:39:24) {
  code: 'ERR_REQUIRE_ESM'
}

Node.js v18.15.0
```

**Comment by jonluca** [comment]
Pushed an update that should fix this, if you want to give it a shot - v1.10.2

**Comment by gzll** [comment]
v1.10.2 works, thank a lot :)
